### PR TITLE
Fixes #4914 - add note about scope

### LINKED ISSUE #4914: Missing information about scopes and how to deal with them
Trying to work with Invoke-Expression I had to do a lot of dig around in order to understand the scope it is running at, would be really nice if we had details about it in this article imo. (was surprised that $private won't hide my vars from being accessed in the invoked expression)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d669d38f-7692-bbaa-2c88-ba1b1d2bb4c1
* Version Independent ID: 9d592d60-e29c-84e6-2e43-078e2a69fb97
* Content: [Invoke-Expression (Microsoft.PowerShell.Utility)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-expression?view=powershell-6#feedback)
* Content Source: [reference/6/Microsoft.PowerShell.Utility/Invoke-Expression.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Utility/Invoke-Expression.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @JamesWTruher
* Microsoft Alias: **jimtru**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes #4914 - introduce remote variables
Fixes [AB#1699107](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1699107)

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.x preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://docs.microsoft.com/powershell/scripting/community/contributing/overview) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md [hunk 1]
```diff
@@ -21,17 +20,31 @@ Invoke-Expression [-Command] <String> [<CommonParameters>]
...
 
 ### Example 1: Evaluate an expression
+
+```powershell
+$Command = "Get-Process"
+$Command
+```
+
+```Output
+Get-Process PS
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md]: Don't think the `PS` is supposed to be there.

File: reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md [hunk 2]
```diff
@@ -42,58 +55,64 @@ Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id   ProcessName
...
+"C:\ps-test\testscript.ps1" | Invoke-Expression
 ```
 
-These commands use **Invoke-Expression** to run a script, TestScript.ps1, on the local computer.
+These commands use `Invoke-Expression` to run a script, TestScript.ps1, on the local computer.
 The two commands are equivalent.
-The first uses the *Command* parameter to specify the command to run.
-The second uses a pipeline operator (|) to send the command string to **Invoke-Expression**.
+The first uses the **Command** parameter to specify the command to run.
+The second uses a pipeline operator (|) to send the command string to `Invoke-Expression`.
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md]: Should the pipeline operator be in a codefence?

Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md]: Fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md (hunk 1 above)
Comment (DCtheGeek) [reply reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md]: This still needs addressing.

Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md]: Oops. Fixed

**Pull request merged**